src/helpers/pressable-functions.php

```diff
@@ -211,9 +215,11 @@ function get_pressable_site_sftp_user_by_id( string $site_id, string $user_id ):
 function get_pressable_site_sftp_user_by_email( string $site_id, string $user_email ): ?object {
 	$sftp_users = get_pressable_site_sftp_users( $site_id );
 
-	foreach ( $sftp_users as $sftp_user ) {
-		if ( ! empty( $sftp_user->email ) && true === is_case_insensitive_match( $sftp_user->email, $user_email ) ) {
-			return $sftp_user;
+	if ( \is_array( $sftp_users ) ) {
```

Comment:
Rather than nesting the logic an extra level down, it may be simpler to just `if ( ! is_array ) { return null; }` and then do the subsequent looping assuming it is an array as we've passed that point?